src/payment/order-details/data/selectors.js

```diff
@@ -0,0 +1,57 @@
+import { createSelector, createStructuredSelector } from 'reselect';
+
+import { getModuleState } from '../../../common/utils';
+import { emailSelector } from '../../../common/selectors';
+import {
+  SEAT_MESSAGE_KEY,
+  CREDIT_SEAT_MESSAGE_KEY,
+  VERIFIED_SEAT_MESSAGE_KEY,
+  SEAT_PRODUCT_TYPE,
+  VERIFIED_CERTIFICATE_TYPE,
+  CREDIT_CERTIFICATE_TYPE,
+  ENROLLMENT_CODE_PRODUCT_TYPE,
+  COURSE_ENTITLEMENT_PRODUCT_TYPE,
+  ENTITLEMENT_MESSAGE_KEY,
+  ENROLLMENT_CODE_MESSAGE_KEY,
+} from './constants';
+
+export const storePath = ['payment', 'basket'];
+
+const paymentSelector = (state, props) =>
+  getModuleState(state, props.storePath ? props.storePath : storePath);
```

Comment:
This is a way of making the selectors more modular.  They have a default expectation of where their data is mounted to the redux store, but it can be overridden/configured via the component's props.